core/preferences/src/main/proto/settings.proto

```diff
@@ -223,11 +223,7 @@ message Settings {
     }
     SearchBarColors color = 3;
     bool launch_on_enter = 4;
-    enum SearchResultOrdering {
-      Alphabetic = 0;
-      Relevance = 1;
-    }
-    SearchResultOrdering search_result_ordering = 5;
+    reserved 5;
```

Comment:
This was never in production (I think?) so no need to reserve this index.

Reply:
this was merged after the last release so yeah probably